integrations-and-supported-tools/great_expectations/notebooks/Neptune_Great_Expectations.ipynb

```diff
@@ -0,0 +1,548 @@
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Expectations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validator.expect_column_values_to_not_be_null(\"pickup_datetime\")\n",
+    "validator.expect_column_values_to_be_between(\"passenger_count\", min_value=1, max_value=6)\n",
+    "validator.save_expectation_suite(\n",
+    "    discard_failed_expectations=False,\n",
+    "    discard_catch_exceptions_kwargs=False,\n",
+    "    discard_include_config_kwargs=False,\n",
+    "    discard_result_format_kwargs=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Log Expectations to Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expectation_suite = validator.get_expectation_suite().to_json_dict()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/meta\"] = expectation_suite[\"meta\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/expectations/expectations_suite_name\"] = expectation_suite[\"expectation_suite_name\"]\n",
+    "\n",
+    "for idx, expectation in enumerate(expectation_suite[\"expectations\"]):\n",
+    "    run[\"gx/expectations\"][idx] = expectation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above code cell logs the GX Expectations suite name to the context configuration to the `gx/expectations/expectations_suite_name` field of the run, and creates a numbered folder for each expectation in the `gx/expectations` namespace."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Checkpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = context.add_or_update_checkpoint(\n",
+    "    name=\"my_quickstart_checkpoint\",\n",
+    "    validator=validator,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Log Checkpoint config to Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/checkpoint/config\"] = stringify_unsupported(checkpoint.config.to_json_dict())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the checkpoint configuration contains lists as values, we use [`stringify_unsupported()`](https://docs.neptune.ai/api/utils/#stringify_unsupported) to convert it to a string."
```

Comment:
```suggestion
    "Since the checkpoint configuration contains lists as values, we use [`stringify_unsupported()`](https://docs.neptune.ai/api/utils/#stringify_unsupported) to convert them to strings."
```
Is it the lists that are unsupported?